frontend/src/components/landing/BundleBanner.tsx

```diff
@@ -0,0 +1,134 @@
+import bundleLogo from "./images/vpn-and-relay-logo.svg";
+
+type FloatingFeaturesProps = {
+  icon: string;
+  text: string;
+  position: string;
+};
+
+export type Props = {
+  runtimeData: RuntimeData;
+};
+
+const FloatingFeatures = (props: FloatingFeaturesProps) => {
+  return (
+    <div
+      className={`${styles[props.position]} ${styles["float-features-item"]}`}
+    >
+      <img alt={props.text} src={props.icon} />
+      <span className={styles["float-features-text"]}>{props.text}</span>
+    </div>
+  );
+};
+
+export const BundleBanner = (props: Props) => {
+  const { l10n } = useLocalization();
+
+  const mainImage = (
+    <img
+      src={womanInBanner.src}
+      alt="Woman in Banner"
+      className={styles["main-image"]}
+    />
+  );
+
+  return (
+    <div className={styles["bundle-banner-wrapper"]}>
+      <div className={styles["first-section"]}>
+        <div className={styles["main-img-wrapper"]}>{mainImage}</div>
+        <div className={styles["float-features-wrapper"]}>
+          <FloatingFeatures
+            icon={bundleFloatOne.src}
+            text={l10n.getString("bundle-feature-one")}
+            position="feature-one"
+          />
+          <FloatingFeatures
+            icon={bundleFloatTwo.src}
+            text={l10n.getString("bundle-feature-two")}
+            position="feature-two"
+          />
+          <FloatingFeatures
+            icon={bundleFloatThree.src}
+            text={l10n.getString("bundle-feature-three")}
+            position="feature-three"
+          />
+        </div>
+      </div>
+      <div className={styles["second-section"]}>
+        <div className={styles["bundle-banner-description"]}>
+          <h2>
+            <Localized
+              id={"bundle-banner-header"}
+              elems={{
+                "vpn-logo": <VpnWordmark />,
+              }}
+            >
+              <span className={styles["headline"]} />
+            </Localized>
+          </h2>
+          <h3>{l10n.getString("bundle-banner-subheader")}</h3>
+          <p>{l10n.getString("bundle-banner-body")}</p>
+          <Localized
+            id={"bundle-banner-1-year-plan"}
+            elems={{
+              b: <b />,
+            }}
+          >
+            <p />
+          </Localized>
+          <div className={styles["pricing-logo-wrapper"]}>
+            <div className={styles["pricing-wrapper"]}>
+              <Localized
+                id={"bundle-price-monthly"}
+                vars={{
+                  // change to getBundlePrice(props.runtimeData)
+                  monthly_price: "$4.99",
+                }}
+                elems={{
+                  "monthly-price": <p className={styles["price"]} />,
+                }}
+              >
+                <span />
+              </Localized>
+              <Localized
+                id={"bundle-price-save-amount"}
+                vars={{
+                  savings: "50%",
+                  old_price: "$11.99",
```

Comment:
Hmm, could you keep these as clearly invalid values for now, to make sure we don't accidentally ship incorrect values to production?

```suggestion
                  savings: "??%",
                  old_price: "$??.??",
```

Reply:
done.